guidelines/guidelines_django.md

```diff
@@ -0,0 +1,68 @@
+# Services Architecture
+
+We use Django to write the backend of our projects and we want to use its functionalities as much as we can to assist us getting the job done. Django has a very opinionated way on how things should be done and this is good. But one thing that it does not gets in the way is how and where to place business logic in your project. 
```

Comment:
I think there's a typo here:

" But one thing that it does not **get** in the way **of** is how and where to place business logic in your project. "